app/megatron/commands/command_actions.py

```diff
@@ -370,4 +370,16 @@ def _change_pause_state(
         ActionType.POST_MESSAGE, {"channel": channel, "message": msg}
     )
     integration_connection.take_action(message_action)
+
+    user_msg = {
+        'text': f"*WARNING:* The bot has been *{paused_word}*.",
+        'attachments': [
+            {
+                "text": "",
+                "footer": f"Executed by Teampay.",
+            }
+        ]
+    }
```

Comment:
I think the *WARNING* language is a little too scary. Maybe something like
> Hey, there! I've been paused so a support agent can talk with you. I'll let you know when I'm back online.

And the footer should be the Agent that paused megatron (like it is for the messages)